test/system/post_test.rb

```diff
@@ -0,0 +1,32 @@
+require "application_system_test_case"
+# https://guides.rubyonrails.org/testing.html#implementing-a-system-test
+
+class PostTest < ApplicationSystemTestCase
+
+  def setup
+    activate_authlogic
+  end
+
+  test 'posting from the editor' do
+    visit '/'
+
+    click_on 'Login'
```

Comment:
1. Here we are visiting login but on line 29 we get the message
``User was successfully created.`` Can you please explain this code snippet?
2. We have same integration test here https://github.com/publiclab/plots2/blob/856c371a5bbba75d9838ca27f4fa6cf0f0c59ece/test/integration/login_flow_test.rb#L4-L13 so one of these will become redundant.
3. In theory of Software Testing at my university, we are taught that System tests are written to `validate the non-functional parameters` like server load time, access time between end nodes etc. Are we also going to do those amazing stuff later on?
4. I am having difficulty in installing the prerequisites for System tests. Can you please tell a good reference? I searched a lot but none worked during past one week.
5. How can we use Recaptcha  or Spamaway in system test? Any idea. You may require them in this pr too in case we want ``assert_selector('#notice', 'User was successfully created.')``.

Reply:
I agree on redundant, but I'm not sure how to create a session we can use to test logged-in stuff. I tried doing as we do in other tests, with `UserSession.new()` or whatever, but I think it literally isn't connecting that session we make in the back-end to an actual logged in session that Capybara can interact with on headless chrome! So, i'm trying to log in manually here... how else ought we to log in? Thanks!

Reply:
> Are we also going to do those amazing stuff later on?

My biggest interests in system tests are:

1. testing the interaction of our complex JS with the Rails app, end-to-end, because things keep breaking here
2. seeing screenshots!

But yeah... lots of other possibilities.